packages/aws-cdk-lib/aws-cloudfront/lib/origin-access-control.ts

```diff
@@ -53,16 +53,9 @@ export enum AccessLevel {
 }
 
 /**
- * Properties for creating a Origin Access Control resource.
+ * Properties for creating a S3 Origin Access Control resource.
  */
-export interface S3OriginAccessControlProps extends OriginAccessControlBaseProps {
-  /**
-   * The level of permissions granted in the bucket policy and key policy (if applicable)
-   * to the CloudFront distribution.
-   * @default AccessLevel.READ
-   */
-  readonly originAccessLevels?: AccessLevel[];
```

Comment:
Removing because I realized this prop is not used and is defined in `S3BucketOriginWithOACProps` ([here](https://github.com/aws/aws-cdk/blob/gracelu0/s3-oac-l2/packages/aws-cdk-lib/aws-cloudfront-origins/lib/s3-bucket-origin.ts#L40)) which makes sense because that's when we need to update the bucket policy.